zeebe/broker/src/main/java/io/camunda/zeebe/broker/bootstrap/JobStreamServiceStep.java

```diff
@@ -134,7 +139,8 @@ private record ImmutableJobActivationPropertiesImpl(
       DirectBuffer worker,
       long timeout,
       Collection<DirectBuffer> fetchVariables,
-      Collection<String> tenantIds)
+      Collection<String> tenantIds,
+      Map<String, Object> authorizations)
```

Comment:
💭 I'm wondering if `authorizations` is the right term here. At first, they are just _claims_, only later we resolve them to authorizations.

Reply:
Yes, technically it's just claims. We name them `authorizations` in the command records, so I was aiming at consistency.

I'll rename it to `claims` for correctness.